frontend/src/components/ApprovalRequests.vue

```diff
@@ -0,0 +1,59 @@
+<template lang="pug">
+  div
```

Comment:
use `template` keyword instead of `div` or nest in some other tag. Currently, the render fails.

Reply:
Done